user_guide_src/source/libraries/throttler.rst

```diff
@@ -117,9 +117,13 @@ filter::
 Next, we assign it to all POST requests made on the site::
 
     public $methods = [
-        'post' => ['throttle', 'csrf'],
+        'post' => ['throttle'],
     ];
 
+.. Warning:: If you use ``$methods`` filters, you should :ref:`disable auto-routing <use-defined-routes-only>`.
+    Because auto-routing permits any HTTP method to access a controller.
```

Comment:
```suggestion
.. Warning:: If you use ``$methods`` filters, you should :ref:`disable auto-routing <use-defined-routes-only>`
    because auto-routing permits any HTTP method to access a controller.
```